src/app/auth/error/page.tsx

```
@@ -0,0 +1,193 @@
+          suggestion: "Verifique suas informações e tente novamente.",
+        };
+      case "SessionRequired":
+        return {
+          title: "Sessão Necessária",
+          description: "Você precisa estar logado para acessar esta página.",
+          suggestion: "Faça login para continuar.",
+        };
+      case "Default":
+      default:
+        return {
+          title: "Erro de Autenticação",
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)